### ISSUE TITLE
Request for revision of bone marrow cancer children

### ISSUE THREAD

**Issue by kkrysiak** [DiseaseOntology/HumanDiseaseOntology#113]
I want to preface my long-winded description with my desire to improve DiseaseOntology and in no way mean this to be taken in a negative way. 

I believe the bone marrow cancer[4960] associations need to be reworked. Currently there are several inappropriate children of myeloma[0070004] and labels that are not accurate. 

1) Children inappropriately assigned to myeloma. Chronic myeloproliferative disease[2226] and myelofibrosis[4971] affect myeloid lineage cells, not plasma cells (lymphoid lineage) as in the definition provided for a myeloma. I'm not as familiar with the other diseases in this category but I believe they should also be checked.
2) Definitions for these diseases should be updated. Myeloma is used in the definition of chronic myeloproliferative disease[2226] and myelofibrosis[4971] but the definition source pages cited for these entries have no mention of the word myeloma. 

I'm not sure how best to go about such a change. However, I can definitely recommend the WHO classification of myeloid neoplasms and acute leukemia as a great source to help with these changes. http://www.ncbi.nlm.nih.gov/pubmed/19357394 Table 2 of this paper shows a hierarchy/associations of myeloid neoplasms that is clinically relevant and one of the standards used by oncologists. The best source of this information could be found in the book "World Health Organization Classification of Tumours of Haematopoietic and Lymphoid Tissues" (4th edition) but I don't know if that is conveniently accessible to you. I understand that this may not be perfect for the sort of organization used in DiseaseOntology, but should still provide a nice guideline. The book also includes lymphoid neoplasms (including plasma cell myeloma) and these have also been described in another publication http://www.ncbi.nlm.nih.gov/pubmed/21300984. Additionally, a heirarchy has been proposed for lymphoid neoplasms with epidemiology in mind which I believe would be very in line with DiseaseOntology. http://www.ncbi.nlm.nih.gov/pubmed?term=17389762 

The complicated relationships between these disorders make designing a hierarchy for them difficult but hopefully the sources I have provided can provide some direction for their improvement in DiseaseOntology.

**Comment by elviram** [comment]
Hello Kilannin,

thank you for your comments and for the time you spend researching this. It is definitely very helpful and we will look into bone marrow cancer branch. It will take some time to get through the sources you are mentioning, but we will keep you informed on our progress and might send some questions your way.

Best,
Elvira

**Comment by lschriml** [comment]
Hello Kilannin,
     thank you for these sources, they are great for figuring out this complex classification. 
I have updated what I can, based on the sources, some of which, as you mentioned, are conflicting. 
I have updated the definition of myeloma.
And updated it's name from myeloma to myeloid neoplasm. 

I also came across a 2016 WHO revision: 
http://www.bloodjournal.org/content/127/20/2391
PDF: http://www.bloodjournal.org/content/bloodjournal/127/20/2391.full.pdf?sso-checked=true

I also reviewed and moved/obsoleted some old terms that were in this area of DO. 

I am going to hold off on further changes as we are working to revise this area of DO this spring. 
I also want to work through the WHO 2016 updates. 

Cheers,
Lynn
